pom.xml

```diff
@@ -185,11 +167,6 @@
                 <artifactId>mockito-core</artifactId>
                 <version>${global_mockito_version}</version>
             </dependency>
-            <dependency>
-                <groupId>org.apache.storm</groupId>
-                <artifactId>storm-kafka</artifactId>
```

Comment:
I don't understand why `storm-kafka` was defined as a dependency here originally.  After removing it, I compared all the dependencies that maven reports via `dependency:tree` and nothing changed.

Reply:
I'm not sure why we have it here, but it's not a dependency directly. It's in the dependencyManagement section, so it's theoretically defining a dependency that children can use without having to specify the version (i.e. instead of the various global params we use, you could also define it here, and the children would just say the group/id). I believe it can also be useful for doing things like moving exclusions to be declared in a single place (e.g. always removing logging deps), but my recollection is a bit hazy.

If it were actually used however, the build would have broken. I have no idea why we have the section in general, because we don't actually use it the way it would be intended.

And I'm also scared to look at the git blame, because there's an extremely nonzero chance I did it.

Reply:
@justinleet @nickwallen - nope, looks like that was me - https://github.com/mmiklavc/metron/blame/7f1160401897c49738dcb79609661419d39e112d/pom.xml#L116. The way the dependency management section works is to lay down versions and other details about a dependency so you can include it in your child projects without having to explicitly specify the version. Ideally, this would be the way we declare all shared dependencies in Metron. Submodules are free to override the default if/when they choose by explicitly declaring the version in their pom dep.

Reply:
If this is not needed, then I think it's fine to remove it. We might consider a separate task at some point of refactoring our poms to use this idiom rather than having to leverage version properties all over. It should clean up our poms considerably.